### DISCUSSION TITLE
I hope this isn't a dumb question but...

### DISCUSSION THREAD

**Discussion by 15Tails** [Techokami/SonicWorldsNext#196] [Q&A] [answered]
I've been wanting to make a 2D Sonic game for years but couldn't for a lot of reasons. (Outdated engines, Incompatibility, I cannot run unreal engine 5, Etc.) Anyway I just discovered this one a couple weeks ago. And I was just wondering.

Is this Engine available for anybody to use to make Sonic Fan games? Or is not just yet completed enough to be used. This is the best 2D Engine I've seen so far and was extra stoked to see it made in godot!

But was wondering if it was possible to make Sonic Fan games in this? 

Also I am very new to GitHub so apologies if I posted this in the wrong location.

**Comment by Candescence** [comment]
I can't speak for the developers, but from what I can tell it _should_ be in a good enough state to actually build upon for actual fangames. Though I do wish there was a list of stuff that still needs to be implemented so people know what's missing, and the external documentation/wiki needs a _lot_ more work.

**Comment by Caverns4** [comment, marked as answer]
You can absolutely use the engine its current state. There's not a ton of documentation, and you do need to have an understanding of Godot somewhat, but this engine is fairly feature-complete, and runs fairly accurately.

**Comment by 15Tails** [comment]
That's incredibly exciting to know! I've been one of the many who was interested in game development since Unity's whole annual fee drama and such. and interest switched over to Godot. I'm still learning it. but I know some semi-basic stuff about it. but hey that's what learning's all about.

also apologies for the incredibly late response... I Was Playing Sonic Adventure 2 Battle Earlier trying to get all 180 emblems. I came with about 164? (Could be mistaken, but I got all "A" ranks for Sonic, shadow, Knuckles and tails and Eggman) Then I hit a brick wall with route 280 as Rouge.... (Don't hit cars and Don't hit Walls is what I got stuck at) I did finally manage the A rank. but ultimately stopped there.

I hope this gets continued updates because it's the first Sonic Engine I'm aware of made in Godot! others I've seen were in Unity, Game maker, and Unreal engine 5.
